test/suite/test_error_info.py

```diff
@@ -26,15 +26,72 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
+import errno
 import wiredtiger, wttest, time
 from wtdataset import SimpleDataSet
+from compact_util import compact_util
+
+import unittest
```

Comment:
We don't need all those packages:

```suggestion
import errno
import wiredtiger
from compact_util import compact_util
```